### DISCUSSION TITLE
Follow-Up on Snap Submission for Rentality

### DISCUSSION THREAD

**Discussion by Badk0m** [MetaMask/snaps#2976] [General]
Hey team,

We submitted the Snap application form for Rentality through your provided Google Form (https://docs.google.com/forms/d/e/1FAIpQLSeDAnk0Di_k8Qgrsj9_ofnf-1d_NfbG3fmSOHD1m8dKun1pXw/viewform)) as outlined in your documentation (https://docs.metamask.io/snaps/how-to/get-allowlisted/)).

The application was submitted by our developer, D--- Vl--- (email: da----a1@gmail.com), and the contact email for the project is 
in----ntality.xyz.

We’d like to kindly follow up on the status of our submission and inquire if there is any additional information required to proceed with the allowlist review process.

Additionally, r055r from Discord recommended that we create this ticket here to follow up on our submission.

Thank you in advance for your support, and we’re looking forward to hearing from you.

Best regards,
Alex
Rentality Team

**Comment by Montoya** [comment]
Thanks for reaching out Alex. We will get it processed this week.

**Reply by Badk0m** [reply]
Thank you, let me know if you need my help. I look forward to your response.

**Reply by Badk0m** [reply]
Thanks, everyone **Flo from Metamask** helped close this topic
https://snaps.metamask.io/snap/npm/rentality/
